test/externalTests/solc-js/solc-js.sh

```diff
@@ -36,7 +36,7 @@ function solcjs_test
     SOLCJS_INPUT_DIR="$TEST_DIR"/test/externalTests/solc-js
 
     # set up solc-js on the branch specified
-    setup_solc "$DIR" solcjs "$SOLJSON" master solc/ "$SOLCJS_CHECKOUT"
+    setup_solc "$DIR" solcjs "$SOLJSON" fix_for_nlohmann_json solc/ "$SOLCJS_CHECKOUT"
```

Comment:
TODO: change back to master once https://github.com/ethereum/solc-js/pull/729 was merged to `master`.